scripts/cleanup_old_vms.py

```diff
@@ -134,21 +97,54 @@ def scan_provider(provider_key, matchers, match_queue, scan_failure_queue):
         return
 
     text_matched_vms = [vm for vm in vm_list if match(matchers, vm.name)]
-    for vm in text_matched_vms:
-        match_queue.put(VmProvider(provider_key, vm.name))
 
-    non_text_matching = set(vm_list) - set(text_matched_vms)
-    logger.info(
-        '%r: NOT matching text filters: %r', provider_key, [vm.name for vm in non_text_matching])
-    logger.info(
-        '%r: MATCHED text filters: %r', provider_key, [vm.name for vm in text_matched_vms])
+    logger.info('%r: NOT matching text filters: %r',
+                provider_key,
+                set([v.name for v in vm_list]) - set([v.name for v in text_matched_vms]))
```

Comment:
{for in} ?

Reply:
not sure what you're getting at here - I'm making two lists of VM names, subtracting the matched names from the full name list in order to display what did not match the given text filter.  This is so on output we can see if there's some problem with the text matching.

I use set() to make sure I end up with unique names, since some providers can have duplicate vm names.